mappings/net/minecraft/entity/LivingEntity.mapping

```diff
@@ -106,17 +106,17 @@ CLASS aiu net/minecraft/entity/LivingEntity
 		ARG 1 jumping
 	METHOD b heal (F)V
 		ARG 1 amount
-	METHOD b getStackInHand (Lahf;)Lbce;
+	METHOD b getItemInHand (Lahf;)Lbce;
```

Comment:
Should this change to `getHeldItem` in line with the setter?

Reply:
Yes, but I don't think this will be merged before the Intermediary move